GETTING_STARTED.md

```diff
@@ -0,0 +1,416 @@
+gitopsconfig-hello-world-yaml-ua5y7o-fs7k9   0/1     Completed   0          6m4s
+hello-world-7b7f58765c-4wtnn                 1/1     Running     0          5m33s
+hello-world-7b7f58765c-7gn4d                 1/1     Running     0          7s
+hello-world-7b7f58765c-wf47l                 1/1     Running     0          7s
+```
+
+Eunomia has reconciled the GitOpsConfig - you uploaded the new version which triggered a new reconciliation cycle in which Eunomia compared the old version of GitOpsConfig with the new one and adjusted resources to reflect it.
+
+### Deploy the third version of GitOpsConfig
+
+```shell
+kubectl apply -f examples/hello-world-yaml/cr/hello-world-cr3.yaml -n eunomia-hello-world-yaml-demo
+```
+
+We can check our test namespace:
+
+```shell
+kubectl get pods -n eunomia-hello-world-yaml-demo
+```
+
+Three new pods should supersede the old ones:
+
+```shell
+NAME                                         READY   STATUS      RESTARTS   AGE
+gitopsconfig-hello-world-yaml-1nhgz3-5t7d7   0/1     Completed   0          6m49s
+gitopsconfig-hello-world-yaml-6qqkns-kpt6m   0/1     Completed   0          46s
+gitopsconfig-hello-world-yaml-ua5y7o-fs7k9   0/1     Completed   0          12m
+hello-world-5df998b7bb-6zw99                 1/1     Running     0          33s
+hello-world-5df998b7bb-744w4                 1/1     Running     0          29s
+hello-world-5df998b7bb-r4zz9                 1/1     Running     0          31s
+```
+
+You should be able to see the `Version: 2.0.0` now:
+
+```shell
+minikube service hello-world -n eunomia-hello-world-yaml-demo
+```
+
+### Clean-up
+
+In the end just delete test and eunomia namespaces:
+
+```shell
+kubectl delete ns eunomia-hello-world-yaml-demo
+kubectl delete ns eunomia-operator
+```
+
+## Hello-world example with a webhook trigger
+
+In this example you'll be using your fork of the Eunomia repository [created earlier](#Checkout-your-fork). You will expose Eunomia's webhook port to the Internet, and configure a webhook on your remote fork to send messages to it on push event.
+
+Contrary to the previous example, Eunomia will be run locally as a binary on your machine. It will watch for GitOpsConfigs in your Minikube cluster.
+
+### Create a new namespace
+
+```shell
+kubectl create namespace eunomia-hello-world-yaml-demo
+```
+
+### Create a new service account
+
+```shell
+kubectl apply -f examples/hello-world-yaml/eunomia-runner-sa.yaml -n eunomia-hello-world-yaml-demo
+```
+
+### Run eunomia binary locally to watch for GitOpsConfigs in Minikube
+
+Use the helper script in a new console window (it will block). Eunomia logs will be printed to the console:
+
+```shell
+./scripts/run-eunomia-locally.sh
+```
+
+If you get any prompts asking to allow application `eunomia-local` to accept incoming network connections, agree.
+
+### Apply GitOpsConfig watching your remote branch
+
+```shell
+cat <<EOF | kubectl apply -n eunomia-hello-world-yaml-demo -f -
```

Comment:
Adding the cat here just seems to complicate the instructions

Reply:
I intended this to be a code snippet for the user to just copy and paste to his own terminal. I also wanted the manifest to be visible in the GETTING_STARTED in order to have everything in one place.
Loading the manifest from stdin seemed most suitable for this example: https://kubernetes.io/docs/reference/kubectl/cheatsheet/#creating-objects